### PR TITLE
feat(helm) add 'get --template' to retrieve details about chart (ie version)

### PR DESCRIPTION
**Pull request by jeff-knurek**
Signed-off-by: Jeff Knurek <j.knurek@travelaudience.com>

**What this PR does / why we need it**:
Adding a `--template` option to `helm get` to be able to obtain the chart version of the release that was installed. 

All other locations the chart version is listed, it gets appended to the chart name, whereas this provides just the version of the chart itself _(and therefor can be used in other helm commands that take `--version`)_

**If applicable**:
- [x] this PR contains documentation 
- [x] this PR contains unit tests
- [x] this PR has been tested for backwards compatibility

### CODE REVIEW

**Comment by bacongobbler** [comment]
I think this would be better suited as a `--output` flag for `helm list`, as the chart metadata is intended to be displayed as part of `helm list`.


For example, if you were to introduce a Go template output format to `helm list`, you could accomplish the same functionality with

```
$ helm list my-release-name --format="{{ .Version }}"
```

That way we avoid the need for introducing commands for things like `helm get home`, `helm get description`, and `helm get maintainers`, for example.

**Comment by jeff-knurek** [comment]
ok, that seems reasonable - but I don't have the slightest clue how to handle using a Go template to select the output

as in, I see where the `type listRelease struct` is currently used to show the output, but....???

also, there's a `.Version` that gets the revision, and a `.Version` that gets the metadata chart version, so what structure would the go template take??

**Comment by jeff-knurek** [comment]
maybe the best compromise is to change this PR to be `helm get metadata --fields Version my-release-name` and then implement the different options in the metadata struct?

**Comment by bacongobbler** [comment]
Have a look at the `helm version --template` flag; that should get you going.

**Comment by jeff-knurek** [comment]
@bacongobbler I had a look into how the `--template` works, and then looked into adding it to `helm list`, however I feel that `helm list` is the wrong place to add this, so I added it to `helm get` directly which I think is definitely better than the original PR 

If you want, I can close this PR and open another PR of just this change

File: docs/helm/helm_get_version.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+## helm get version
```
Comment (hickeyma) [inline docs/helm/helm_get_version.md]: This file is not needed and should be removed. All docs in docs/helm are generated file from the text specified in the helm command definition.  Ref: https://helm.sh/docs/developers/#man-pages

**Review by hickeyma — CHANGES_REQUESTED** [review]

**Review by hickeyma — CHANGES_REQUESTED** [review]
@jeff-knurek Thanks for working on this.

I have tested it and it makes sense to be part of `helm get` because you want to get the chart version for a release.

Tested as follows:
```console
$ helm ls
NAME     	REVISION	UPDATED                 	STATUS  	CHART          	APP VERSION	NAMESPACE
chrt-5586	2       	Wed May  1 17:23:18 2019	DEPLOYED	chrt-5586-0.1.0	1.0        	default  

$ helm get chrt-5586 --template="{{ .Release.Name }}" 
chrt-5586

$ helm get chrt-5586 --template="{{ .Release.Chart.Metadata.Version }}" 
0.1.0
```

Can you remove `docs/helm/helm_get_version.md` as it is not required.

**Comment by jeff-knurek** [comment]
ok, file removed and I updated the title/description of the PR

**Review by hickeyma — APPROVED** [review]
LGTM, thanks for this.

**Pull request merged**